### PR TITLE
[Term Entry] PHP Math Functions sin()

### PR DESCRIPTION
**Pull request by jorgeprietoarranz**
### Description

- Added the sin() term for PHP
- Closes Issue #2059 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/php/concepts/math-functions/terms/sin/sin.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  - 'Computer Science'
+  - 'Web Development'
+Tags:
+  - 'Arithmetic'
+  - 'Functions'
+  - 'Methods'
+  - 'Numbers'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
```
Comment (SSwiniarski) [inline content/php/concepts/math-functions/terms/sin/sin.md:14]:
```suggestion
```
We generally limit CatalogContent to two entries.

File: content/php/concepts/math-functions/terms/sin/sin.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  - 'paths/front-end-engineer-career-path'
+---
+
+The **`sin()`** function returns the sine of a given angle in radians.
```
Comment (SSwiniarski) [inline content/php/concepts/math-functions/terms/sin/sin.md:18]:
```suggestion
The **`sin()`** function returns the sine of a angle given in radians.
```
Original wording can read as if the result was given in radians.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@jorgeprietoarranz I did a first review and left a couple of comments.

File: content/php/concepts/math-functions/terms/sin/sin.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+  - 'Arithmetic'
+  - 'Functions'
+  - 'Methods'
+  - 'Numbers'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The **`sin()`** function returns the sine of a angle given in radians.
```
Comment (SSwiniarski) [inline content/php/concepts/math-functions/terms/sin/sin.md:17]:
```suggestion
The **`sin()`** function returns the sine of an angle given in radians.
```

**Review by SSwiniarski — APPROVED** [review]
@jorgeprietoarranz Other than one line edit, looks good for a second review.

**Review by KTom101 — CHANGES_REQUESTED** [review]
@jorgeprietoarranz I've added a second review to this entry.

**Comment by jorgeprietoarranz** [comment]
> @jorgeprietoarranz I've added a second review to this entry.

Thanks a lot for your suggestions!

**Review by KTom101 — APPROVED** [review]
@jorgeprietoarranz Added a minor change in the codebyte so we can have an example with a different output. I also ran `npm run format:all` to fix the tests failing. You can read more on that [here](https://github.com/Codecademy/docs/blob/main/.github/CONTRIBUTING.md#how-can-i-test-my-changes-before-submitting-a-pr).

**Comment by jorgeprietoarranz** [comment]
I have just followed the steps, so my contribution should not appear as anonymous anymore.

Thanks a lot for the experience and hope see you around soon ^_^

**Pull request merged**